File: neatformcore/src/main/java/com/nerdstone/neatformcore/form/json/NFormViewBuilder.java
package com.nerdstone.neatformcore.form.json;

import android.view.View;

import com.nerdstone.neatformcore.utils.Constants;

import java.util.Map;

public class NFormViewBuilder {
    public static void makeView(Map<String, Object> attributes, View view, String type) {
        switch (type) {
            case Constants.ViewType.EDIT_TEXT:
                makeEditText(attributes, view);
                break;
        }
    }

    private static void makeEditText(Map<String, Object> attributes, View view) {

    }


}

File: neatformcore/src/main/java/com/nerdstone/neatformcore/form/json/NFormViewBuilder.kt
package com.nerdstone.neatformcore.form.json

import android.view.View

import com.nerdstone.neatformcore.utils.Constants

object NFormViewBuilder {

    fun makeView(attributes: Map<String, Any>, view: View, type: String) {
        when (type) {
            Constants.ViewType.EDIT_TEXT -> makeEditText(attributes, view)
        }
    }

    private fun makeEditText(attributes: Map<String, Any>, view: View) {

    }

}
